client/layout/masterbar/blaze-pro.scss

```diff
@@ -49,20 +49,16 @@ body.is-section-signup {
 	--color-white: #fff;
 	--color-black: $colorBlack;
 	--color-gray: #50575e;
-	--color-orange: #ff6433;
```

Comment:
Moved to https://github.com/Automattic/wp-calypso/pull/103611/files#diff-f37ca3eedcd3a46382e1bfec60a2ed99d460e9afb111cc1492f29f216c549e8cR156